infra/build/functions/base_images.py

```diff
@@ -22,13 +22,15 @@
 
 import build_base_images
 
+BASE_PROJECT = 'oss-fuzz-base'
+
 
 # pylint: disable=no-member
 def base_builder(event, context):
   """Cloud function to build base images."""
   del event, context
-  credentials, project_id = google.auth.default()
-  tag_prefix = f'gcr.io/{project_id}/'
+  credentials, _ = google.auth.default()
+  tag_prefix = f'gcr.io/{BASE_PROJECT}/'
```

Comment:
Did you forget a .format here ?

Reply:
This is a Python 3 language feature :) https://realpython.com/python-f-strings/